lib/foreman_rh_cloud/settings.rb

```diff
@@ -2,6 +2,7 @@
   category(:rh_cloud, N_('RHCloud')) do
     setting('allow_auto_inventory_upload', type: :boolean, description: N_('Enable automatic upload of your host inventory to the Red Hat cloud'), default: true, full_name: N_('Automatic inventory upload'))
     setting('allow_auto_insights_sync', type: :boolean, description: N_('Enable automatic synchronization of Insights recommendations from the Red Hat cloud'), default: false, full_name: N_('Synchronize recommendations Automatically'))
+    setting('allow_auto_insights_mismatch_delete', type: :boolean, description: N_('Enable automatic deletion of mismatched host records from the cloud'), default: false, full_name: N_('Automatic mismatch deletion'))
```

Comment:
I noticed in the above setting you have `the Red Hat cloud` can we change this line to match the other settings?

Reply:
Lemme change this one